internal/services/compute/managed_disk_resource.go

```diff
@@ -215,6 +215,17 @@ func resourceManagedDisk() *pluginsdk.Resource {
 				ForceNew: true,
 			},
 
+			"hyper_v_generation_version": {
```

Comment:
'generation' implies this is a version - so the `version` suffix here is unnecessary:

```suggestion
			"hyper_v_generation": {
```

Reply:
_version removed